### ISSUE TITLE
Don't stop background music when watching a gif (video without sound)

### ISSUE THREAD

**Issue by kit-ty-kate** [stom79/mastalab#463]
Title is self-explanatory I think ^^

---

Mastalab version: 1.8.9

**Comment by stom79** [comment]
Yes, it is intended, I think people should be fully concentrated when watching a gif. More seriously, it's something I have never changed since a while, I will work on this bug.
